### PR TITLE
Removes reference to "cherry-pick candidate" label, which is no longe…

### PR DESCRIPTION
**Pull request by guineveresaenger**
…r used

### CODE REVIEW

File: release-team/README.md [hunk 1]
```diff
@@ -94,9 +94,8 @@ Release responsibilities of individual contributors to the Kubernetes project ar
 
 #### During a patch release
 If you have a patch that needs to be ported back to a previous release (meaning it is a critical bug/security fix), once it is merged to the Kubernetes `master` branch:
-- Mark your PR with the milestone corresponding to the release you want to port back to (e.g. `v1.5`), and add the `cherrypick-candidate` label to it
-- The Patch Release Manager will then review the PR and if it is ok for cherry-picking, will apply a `cherrypick-approved` label to it
-- Once your PR has been marked with the `cherrypick-approved` label by the Patch Release Manager, you should prepare a cherry-pick to the requested branch following the instructions [here](https://github.com/kubernetes/community/blob/master/contributors/devel/cherry-picks.md#how-do-cherrypick-candidates-make-it-to-the-release-branch)
+- Follow the [cherry-pick instructions to open a cherry-pick PR.](https://github.com/kubernetes/community/blob/master/contributors/devel/cherry-picks.md
```
[the remark this answers is not in the document]
Comment (guineveresaenger) [reply release-team/README.md]: gahhhh too many git stashes
INSTA RESPONSE

[the remark this answers is not in the document]
Comment (nikhita) [reply release-team/README.md]: still needs a closing bracket at the end lol

**Comment by guineveresaenger** [comment]
Apologies - found more outdated references. 😬

File: release-team/README.md [hunk 2]
```diff
@@ -94,9 +94,8 @@ Release responsibilities of individual contributors to the Kubernetes project ar
 
 #### During a patch release
 If you have a patch that needs to be ported back to a previous release (meaning it is a critical bug/security fix), once it is merged to the Kubernetes `master` branch:
-- Mark your PR with the milestone corresponding to the release you want to port back to (e.g. `v1.5`), and add the `cherrypick-candidate` label to it
-- The Patch Release Manager will then review the PR and if it is ok for cherry-picking, will apply a `cherrypick-approved` label to it
-- Once your PR has been marked with the `cherrypick-approved` label by the Patch Release Manager, you should prepare a cherry-pick to the requested branch following the instructions [here](https://github.com/kubernetes/community/blob/master/contributors/devel/cherry-picks.md#how-do-cherrypick-candidates-make-it-to-the-release-branch)
+- Follow the [cherry-pick instructions to open a cherry-pick PR.](https://git.k8s.io/community/contributors/devel/cherry-picks.md
```
Comment (AishSundar) [inline release-team/README.md]: missing )

**Comment by AishSundar** [comment]
Thanks @guineveresaenger. Just 1 nit. Feel free to remove hold after fixing that.

/lgtm
/hold

**Comment by guineveresaenger** [comment]
/remove hold

Fixed missing paren. 🤦‍♀️

**Comment by nikhita** [comment]
Assuming you meant to remove the hold :)

/hold cancel

**Pull request merged**